onnxruntime/core/providers/rocm/ThreadUtil.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstdint>
+
+//namespace onnxruntime {
+
+//namespace profiling {
```

Comment:
>profiling [](http://example.com/codeflow?start=12&length=9)

Uncomment the namespace?
Like discussed - we prefer RocTracer to go with built-in threading utilities. But reallly we have to add it here, let's put them into a namespace.